fileServer.js

```diff
@@ -0,0 +1,48 @@
+ 'use strict';
+
+ var http = require('http');
+ var fs = require('fs');
+ var path = require('path');
+ var url = require('url');
+ var rootDir = __dirname + '/public';
+
+
+ const mimeType = {
+ 	'html': 'text/html',
```

Comment:
Spacing - your development team and mine use the Air BnB javascript style guide 

https://github.com/airbnb/javascript/

Two spaces for indents and please start following other conventions in that guide as you code.